test/IISIntegration.FunctionalTests/NtlmAuthentationTest.cs

```diff
@@ -117,10 +115,6 @@ Task NtlmAuthentication_CoreCLR(RuntimeFlavor runtimeFlavor, RuntimeArchitecture
                         responseText = await response.Content.ReadAsStringAsync();
                         Assert.Equal(HttpStatusCode.OK, response.StatusCode);
                         Assert.NotEmpty(responseText);
-
-                        response = await httpClient.GetAsync("/AutoForbid");
-                        responseText = await response.Content.ReadAsStringAsync();
-                        Assert.Equal(HttpStatusCode.Forbidden, response.StatusCode);
```

Comment:
Autoforbid can also be removed from the test site: https://github.com/aspnet/IISIntegration/blob/c267d192d682b2c22bd38e0c16df21157b10a579/test/TestSites/StartupNtlmAuthentication.cs#L64